docs/sdk/getting-started/cordova.md

```diff
@@ -126,7 +126,7 @@ As a complete example, for a Apache Cordova project that supports both Android a
 
 If you're using App Center Analytics in your app, there's some additional configuration steps you must perform. Open the Apache Cordova project's `config.xml` file in an editor, and add one or more of the following `preferences` elements to the file:
 
-- `APPCENTER_ANALYTICS_ENABLE_IN_JS` - *(optional, default is false)* Controls whether Analytics is enabled automatically. When `false` Analytics is enabled by default. When `true`, the application must call `Analytics.setEnabled()` (in its JavaScript code) to enable Analytics before sending data App Center. This preference is useful when you want to ask users whether they want to share analytics information before enabling it within the application. [Learn more about sending user events manually.](~/sdk/analytics/cordova.md#wait-for-js-to-enable-app-center-analytics)
+- `APPCENTER_ANALYTICS_ENABLE_IN_JS` - *(optional, default is false)* Controls whether Analytics is enabled automatically. When `false` Analytics is enabled by default. When `true`, the application must call `AppCenter.Analytics.setEnabled(true)` (in its JavaScript code) to enable Analytics before sending data App Center. This preference is useful when you want to ask users whether they want to share analytics information before enabling it within the application. [Learn more about sending user events manually.](~/sdk/analytics/cordova.md#wait-for-js-to-enable-app-center-analytics)
```

Comment:
1. "When `false`..." requires a comma after `false` if I'm not mistaken.
2. "before sending data TO App Center."